tools/graph-links

```diff
@@ -95,6 +109,18 @@ TASK_TYPES = {
 logging.basicConfig(format='%(message)s', stream=sys.stdout)
 logger = logging.getLogger()
 
+config = ConfigParser.ConfigParser()
+config.read('/etc/archivematica/MCPClient/archivematicaClientModules')
+
+
+def get_script_name(name):
+    if not isinstance(name, unicode):
```

Comment:
Tip: Better to check for `isinstance(name, basestring)` http://stackoverflow.com/questions/4843173/how-to-check-if-type-of-a-variable-is-string-in-python